lib/rbac/seed.rb

```diff
@@ -0,0 +1,133 @@
+require 'rbac-api-client'
+module RBAC
+  class Seed
+    def initialize(seed_file, user_file)
+      @acl_data = YAML.load_file(seed_file)
+      @request = create_request(user_file)
+    end
+
+    def process
+      ManageIQ::API::Common::Request.with_request(@request) do
+        create_groups
+        create_roles
+        create_policies
+      end
+    end
+
+    private
+
+    def create_groups
+      current = current_groups
+      names = current.collect(&:name)
+      group = RBACApiClient::Group.new
+      begin
+        RBAC::Service.call(RBACApiClient::GroupApi) do |api_instance|
+          @acl_data['groups'].each do |grp|
+            next if names.include?(grp['name'])
+
+            Rails.logger.info("Creating #{grp['name']}")
+            group.name = grp['name']
+            group.description = grp['description']
+            api_instance.create_group(group)
+          end
+        end
+      rescue RBACApiClient::ApiError => e
+        Rails.logger.error("Exception when calling GroupApi->create_group: #{e}")
+        raise
+      end
+    end
+
+    def current_groups
+      RBAC::Service.call(RBACApiClient::GroupApi) do |api|
+        RBAC::Service.paginate(api, :list_groups,  {}).to_a
+      end
+    end
+
+    def create_roles
```

Comment:
why don't you use the `Roles` class you created under the same directory? Same for all other RBAC objects.